Mathlib/Data/Polynomial/Splits.lean

```diff
@@ -464,20 +485,18 @@ theorem prod_roots_eq_coeff_zero_of_monic_of_split {P : K[X]} (hmo : P.Monic)
     congr
     ext
     rw [neg_eq_neg_one_mul]
-  rw [Multiset.prod_map_mul, Multiset.map_const, Multiset.prod_replicate, Multiset.map_id',
-    splits_iff_card_roots.1 hP]
+  simp only [splits_iff_card_roots.1 hP, neg_mul, one_mul, Multiset.prod_map_neg]
```

Comment:
Do you know which step of the `rw` failed?

Reply:
The rewrite fails at `Multiset.map_const`. Like the third remaining error in the file at `image_rootSet`, visually the pattern does match but Lean is not getting it.